pkg/framework/framework.go

```diff
@@ -64,11 +66,18 @@ func NewFramework(userName string) (*Framework, error) {
 		return nil, fmt.Errorf("error when initializing appstudio hub controllers for sandbox user: %v", err)
 	}
 
+	// "pipeline" service account needs to be present in the namespace before we start with creating tekton resources
+	// TODO: STONE-442 - decrease the timeout here back to 30 seconds once this issue is resolved.
+	userNamespace := fmt.Sprintf("%s-tenant", k.UserName)
+	if err = utils.WaitUntil(asAdmin.CommonController.ServiceaccountPresent("pipeline", userNamespace), time.Second*60); err != nil {
+		return nil, fmt.Errorf("'pipeline' service account wasn't created in %s namespace: %+v", userNamespace, err)
+	}
```

Comment:
this should fix the issue seen in [latest CI runs](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/redhat-appstudio_e2e-tests/361/pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e/1628032704769429504/artifacts/redhat-appstudio-e2e/redhat-appstudio-gather/artifacts/pipelineruns.json)